modules/web/src/main/java/com/haulmont/masquerade/components/SideMenu.java

```diff
@@ -0,0 +1,109 @@
+package com.haulmont.masquerade.components;
+
+import com.haulmont.masquerade.Conditions;
+import com.haulmont.masquerade.util.Log;
+
+import java.util.Arrays;
+
+/**
+ * SideMenu component.
+ * <br>
+ * Supported conditions:
+ * <ul>
+ * <li>{@link Conditions#EXPANDED}</li>
+ * <li>{@link Conditions#COLLAPSED}</li>
+ * </ul>
+ */
+public interface SideMenu extends Component<SideMenu> {
+
+    /**
+     * Opens side menu item matching the given {@code screenClass} and {@code path}.
+     * <p>
+     * Example:
+     * <pre>
+     *     $c(SideMenu).openItem(UserBrowser.class, 'administration', 'sec$User.browse')
```

Comment:
Since the snippet is in Groovy, we can omit `.class` part:

`$c(SideMenu).openItem(UserBrowser, 'administration', 'sec$User.browse')`